truss/truss_handle.py

```diff
@@ -1067,7 +1067,7 @@ def _wait_for_docker_build(container) -> None:
 
 
 @retry(
-    stop=stop_after_delay(120),
```

Comment:
@squidarth The motivation is to make trussless integration test fail faster when the container successfully built but failed to start the server, currently we need to wait 120 seconds to see that failure. I want to make sure it's not breaking anything.

Reply:
is this used anywhere outside of the tests?

Reply:
yes, it does look like it is. Could you change this to have the "stop_after_delay" passed as a parameter?

You can switch to using this instead: https://tenacity.readthedocs.io/en/latest/index.html#changing-arguments-at-run-time